src/intl.ts

```diff
@@ -49,6 +49,13 @@ export const commonMessages = defineMessages({
   generalInformations: {
     defaultMessage: "General Information"
   },
+  imgageUploadError_title: {
+    defaultMessage: "Couldn't process image"
+  },
+  imageUploadError_text: {
+    defaultMessage:
+      "There was a poblem with the file you uploaded as an image and it couldn't be used. Please try a different file."
+  },
```

Comment:
I don't think these are common messages. They should be defined somewhere around the component using them.

Reply:
The content has been confirmed internally, and is included in the ticket.

Reply:
`errorMessages` sounds good

Reply:
Changed for the errors I dealt with. Probably a good idea to adjust for other types of errors still in commonMessages.